tests/crypto_test.cc

```diff
@@ -45,6 +46,25 @@ TEST(crypto, sign_verify_rsa_file) {
   EXPECT_TRUE(signe_is_ok);
 }
 
+TEST(crypto, sign_tuf) {
+  std::string private_key = Utils::readFile("tests/test_data/priv.key");
+  std::string public_key = Utils::readFile("tests/test_data/public.key");
+  Config config;
+  TemporaryDirectory temp_dir;
+  config.storage.path = temp_dir.Path();
+  boost::shared_ptr<INvStorage> storage = boost::make_shared<FSStorage>(FSStorage(config.storage));
+  storage->storePrimaryKeys(public_key, private_key);
+  CryptoKey keys(storage, config);
+
+  Json::Value tosign_json;
+  tosign_json["mykey"] = "value";
+  Json::Value signed_json = keys.signTuf(tosign_json);
+  EXPECT_EQ(signed_json["signed"]["mykey"].asString(), "value");
+  EXPECT_EQ(signed_json["signatures"][0]["keyid"].asString(),
+            "6a809c62b4f6c2ae11abfb260a6a9a57d205fc2887ab9c83bd6be0790293e187");
+  EXPECT_EQ(signed_json["signatures"][0]["sig"].asString().size() != 0, true);
+}
+
```

Comment:
BTW I think @lbonn moved this test to the new uptane_crypto_test file.

Reply:
Yes, Thank You.